README.md

```diff
@@ -79,6 +79,13 @@ Amphion provides a comprehensive objective evaluation of the generated audio. Th
 
 Amphion unifies the data preprocess of the open-source datasets including [AudioCaps](https://audiocaps.github.io/), [LibriTTS](https://www.openslr.org/60/), [LJSpeech](https://keithito.com/LJ-Speech-Dataset/), [M4Singer](https://github.com/M4Singer/M4Singer), [Opencpop](https://wenet.org.cn/opencpop/), [OpenSinger](https://github.com/Multi-Singer/Multi-Singer.github.io), [SVCC](http://vc-challenge.org/), [VCTK](https://datashare.ed.ac.uk/handle/10283/3443), and more. The supported dataset list can be seen [here](egs/datasets/README.md) (updating).
 
+### Visualization
+
+Amphion provides visualization tools to interactively illustrate the internal processing mechanism of classic models. This provides an invaluable resource for educational purposes and for facilitating understandable research.
+
+Currently, Amphion supports [SingVisio](https://arxiv.org/abs/2402.12660), a visualization tool of the diffusion model for singing voice conversion. And, SingVisio tool can be experienced [here](https://openxlab.org.cn/apps/detail/Amphion/DiffSVC).
```

Comment:
1. Add some badges to beautify (like arXiv paper badge).
2. The displayed SingVisio URL is openxlab's. Do we need to change it to https://dsvc.openmmlab.org.cn/ ?

Reply:
1. Badges were added in [SingVisio Recipe](https://github.com/open-mmlab/Amphion/pull/141/files#diff-d72f320578d95d54f927d876dcf9c7661f69c197a6838d33f066eab2d33a2dd9R3-R4).
2. SingVisio UR was updated.

Reply:
1. Badges were added in [SingVisio Recipe](https://github.com/open-mmlab/Amphion/blob/6a8e0c0402a7275eb6e56941956b399f4125dfd1/egs/visualization/SingVisio/README.md).
2. New SingVisio URL was upated.